solace-spring-cloud-stream-binder/solace-spring-cloud-stream-binder-core/src/test/java/com/solace/spring/cloud/stream/binder/util/XMLMessageMapperTest.java

```diff
@@ -363,6 +363,8 @@ public void testMapSpringMessageToXMLMessage_NonWriteableSolaceProperties() thro
 				case SolaceBinderHeaders.CONFIRM_CORRELATION:
 					assertNull(xmlMessage.getProperties().get(header.getKey()));
 					break;
+				case SolaceBinderHeaders.NULL_PAYLOAD:
+					break; //Nothing to check as this property is not set on the xmLMessage
```

Comment:
Need to check that the `xmlMessage.getProperties()` is `null`. The purpose of this test is ensure that if an application sets a non-writable header on their outbound Spring message, that it is ignored and doesn't get propagated to the `xmlMessage.getProperties()`.

Reply:
Right, thanks
Done